openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -195,7 +294,7 @@ parameters:
   displayName: Site URL
   description: The URL for the site
   required: true
-  value: https://email-verification.vonx.io
+  value: https://email-verification.apps.silver.devops.gov.bc.ca
```

Comment:
The values for this and `APPLICATION_SERVICE_DOMAIN` should match.

Reply:
done